test/test.js

```diff
@@ -292,16 +293,57 @@ describe('onclickoutside hoc', function() {
   });
 
   describe('with child rendering as null', function() {
-    var StatelessComponent = () => null;
```

Comment:
Hm, aint sure if I follow what u mean exactly - but I'll restore the test with switched asserts in try/catch.